src/capture/es_analyzer.cc

```diff
@@ -0,0 +1,635 @@
+#include "capture/es_analyzer.h"
+
+#include <algorithm>
```

Comment:
Reorder include files in following rules.
- the header file that this .cc implements. i.e. "capture/es_analyzer.h"
- standard headers, i.e. <algorithm> (<casserts> should be here in alphabetical order)
- header files in this repository. i.e. "gui/util.h"
- header files in third_party libraries. i.e. <SDL.h>
Sort files by their paths in alphabetical order in each category.